language/move-analyzer/editors/code/src/context.ts

```diff
@@ -8,12 +8,58 @@ import * as vscode from 'vscode';
 import * as lc from 'vscode-languageclient';
 import { log } from './log';
 
+/**
+ * OnReady The ready event is fired when the language client is ready to accept
```

Comment:
What is the client ready to accept?

Reply:
This comment is not very clear, I will update it